sphinx/source/docs/user_guide/styling.rst

```diff
@@ -1106,6 +1106,29 @@ the render level ``"image"`` to the ``level`` argument when calling your
 You can see a complete example with output in the section
 :ref:`userguide_plotting_images_colormapped`.
 
+.. _userguide_styling_latex:
+
+Adding mathematical notations with LaTeX
+----------------------------------------
+
+Bokeh supports mathematical notations expressed in the LaTeX_ markup language
+with a growing number of plot elements. Currently, you can use LaTeX notations
+only with :ref:`axis labels <userguide_styling_axes_labels>`. Supporting LaTeX
+notations in more elements is planned for future releases.
+
+Bokeh uses the MathJax_ library to handle LaTeX markup. To use LaTeX markup
+instead of plain text, use a :class:`~bokeh.models.math_text.MathText` object:
+
+.. bokeh-plot:: docs/user_guide/examples/styling_axis_labels_math_text.py
+    :source-position: above
+
+.. note::
+    The ``text`` attribute of ``MathText`` only accepts LaTeX notations. You do
+    not need to include any delimiters such as ``$$`` or ``$``.
+
+.. _LaTeX: https://www.latex-project.org/
+.. _MathJax: https://www.mathjax.org
```

Comment:
@tcmetzger should we add these to the global file for these kinds of links?

Reply:
Yes - I'll put that on a separate, rst_epilog-related PR!